docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content_page_macros.twig

```diff
@@ -44,3 +44,36 @@
     {% endif %}
   {% endif %}
 {% endmacro %}
+
+
+{% macro displayPublicUse() %}
```

Comment:
These are going to need translations

Reply:
Actually, these need to be removed. Public use is being handled a different way.

Reply:
There's also blocks for Blog Comment Policy, Syndication Use and Press Contacts. Are these all being handled in a custom manner?